rasa/core/channels/socketio.py

```diff
@@ -155,6 +160,9 @@ def blueprint(
             sio, self.socketio_path, "socketio_webhook", __name__
         )
 
+        # make sio object static to use in get_output_channel
+        SocketIOInput.sio = sio
```

Comment:
IMO we have only one instance of SocketIOInput created, so that doesn't have to be static. 
I think you can use `self.sio` to store it.

Reply:
What about when we use multiple Sanic workers?

Reply:
Shouldn't matter. Each sanic worker uses a different process which means that they don't share any (not process safe) variables anyway.

Reply:
I've updated pull request with the suggestion